governator-jersey/src/main/java/com/netflix/governator/guice/jersey/GovernatorJerseySupportModule.java

```diff
@@ -0,0 +1,80 @@
+package com.netflix.governator.guice.jersey;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.netflix.governator.guice.jetty.GovernatorServletContainer;
+import com.netflix.governator.providers.AdvisableAnnotatedMethodScanner;
+import com.netflix.governator.providers.ProvidesWithAdvice;
+import com.sun.jersey.api.core.DefaultResourceConfig;
+import com.sun.jersey.api.core.PackagesResourceConfig;
+import com.sun.jersey.api.core.ResourceConfig;
+import com.sun.jersey.guice.spi.container.servlet.GuiceContainer;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.servlet.ServletContext;
+
+/**
+ * Module enabling support for Jersey configuration via advisable bindings for {@link DefaultResourceConfig}.
+ * This module provides an alternative Guice integration based on jersey-guice {@link GuiceContainer} but with 
+ * less opinions, such as automatically adding any bound class with {@literal @}Path as a resources.
+ * 
+ * Applications are expected to customize {@link DefaultResourceConfig} to provide a whitelist of
+ * root resources to expose.  This can be done by adding the following {@link @Advises} method.
+ * 
+ * <pre>{@code
+@Advises
+@Singleton
+@Named("governator")
+UnaryOperator<DefaultResourceConfig> adviseWithMyApplicationResources() {
+    return defaultResourceConfig -> {
```

Comment:
This one might be confusing since it will be implicitly done by an internal @Advice that ties properties to archaius.  I'll make it clear that any change to DefaultResourceConfig will be picked up.